zel/src/org/zkoss/zel/ImportHandler.java

```diff
@@ -155,11 +158,25 @@ public void importPackage(String name) {
                         null, "importHandler.invalidPackage", name));
             }
         }
-        packages.add(name);
+        List packageList = importPackages.get(rqpath);
+        if (packageList == null) {
+            packageList = new ArrayList<String>(Arrays.asList(JAVA_LANG_PACKAGE));
+        }
+        if (!packageList.contains(name)) {
+            packageList.add(name);    
+        }
+        importPackages.put(rqpath , packageList);

```

Comment:
See this first - http://tracker.zkoss.org/browse/ZK-3953, the method is simplified. Could have resolved this issue?

Reply:
I think it is not

Reply:
it is still throw exception